documentation/index.html.md

```diff
@@ -0,0 +1,130 @@
+* [Calling model operations](./operations-overview/)
+
+Learn more about advanced topics:
+
+* [Working with templates](#templates)
+* [Understanding channels](./channel-overview/)
+* [Additional options for initializing](./generated/flow-js/)
+
+**The current version of Flow is 0.9.0.** See the [Using Flow.js in your Project](#using_in_project) section below. You can also view the history of releases on <a href="https://github.com/forio/flow.js/releases/" "target=_blank">GitHub</a>.
+
+
+<a name="using_in_project"></a>
+###Using Flow.js in your Project
+
+**To use Flow.js in your project:**
+
+1. Add the Flow.js required libraries to your project. Flow.js requires the following files:
+	* [`jquery.js`](http://jquery.com): document manipulation, DOM element selection, and event handling used by Flow.js
+	* [`lodash.js`](http://lodash.com): utilities and performance enhancements used by Flow.js; also used in [templating](#templates)
+	* [`epicenter.js`](https://forio.com/tools/js-libs/1.5.0/epicenter.min.js): [Epicenter API Adapters](../api_adapters/) with services and utilities for connecting to project models using the underlying Epicenter RESTful APIs.
+2. Add Flow.js itself to your project. The latest version of the Flow.js library is available from our set of tools: <a href="https://forio.com/tools/js-libs/flow/0.9.0/flow.min.js" target="_blank">https://forio.com/tools/js-libs/flow/0.9.0/flow.min.js</a>. (You can also review previous versions and detailed release notes on <a href="https://github.com/forio/flow.js/releases" target="_blank">GitHub</a>.)
+3. Call the `Flow.initialize()` method. This tells Flow.js to create and initialize a run for you. (Runs are sets of particular user interactions with your project.)
+4. In order to finish initializing a run, Flow.js needs to know the name of the model. Add the attribute `data-f-model` to the `<body>` tag. Set the value to the name of your [model file](../writing_your_model/).
+
+**Example:**
+
+		<html>
+			<head>
+				<script src="yourPath/jquery.min.js"></script>
+				<script src="yourPath/lodash.js"></script>
+				<script src="yourPath/epicenter.min.js"></script>
+				<script src="yourPath/flow.js"></script>
+				
+				<script>
+				$(function() { Flow.initialize(); });
+				</script>
+			</head>
+			<body data-f-model="model.vmf">
+
+			</body>
+		</html>
+
+
+**Notes:**
+
+The `Flow.initialize()` call optionally takes an argument, `channel`, which includes a `run`.
+
+The `run` is an object that includes:
+	
+* `account`: Optional, **User ID** or **Team ID**
+* `project`: Optional, **Project ID**
+* `model`: Optional, name of primary model file (repeated from `data-f-model` in your HTML `<body>` tag)
+
+These parameters are optional because the `run` information defaults correctly based on the location of your user interface file within your Epicenter project and the `data-f-model` element in your `<body>` tag. But you can add it if desired: 
+
+		Flow.initialize({
+			channel: {
```

Comment:
link to model run params from here? i.e., to show that you can also pass in `scope`, `files` etc